src/calliope/backend/where_parser.py

```diff
@@ -282,42 +304,49 @@ def __init__(self, instring: str, loc: int, tokens: pp.ParseResults) -> None:
         self.loc = loc
 
     def __repr__(self):
+        """Return string representation of the parsed grammar."""
         return f"SUBSET:{self.set_name}{self.val}"
 
     def _eval(self) -> list[str | float]:
-        "Evaluate each element of the subset list"
+        """Evaluate each element of the subset list."""
         values = [val.eval("array", **self.eval_attrs) for val in self.val]
         return [val.item() if isinstance(val, xr.DataArray) else val for val in values]
 
     def as_math_string(self) -> str:
+        """Process and return as LaTeX."""
```

Comment:
As with the backend models, these docstrings are inherited from the ABC so they shouldn't need to be repeated ad nauseam. We need a solution for this on the linting side

Reply:
Gah, you are right. I wish we could just use `@override` (https://docs.python.org/3/library/typing.html#typing.override), then the linter would catch these cases.

Unfortunately it would break compatibility with 3.11 and lower :(

Reply:
I'll just add   # noqa: D102, override. Already did it for other functions, it'll be enough for now.